spec/factories/notifications.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :notification do
+    status "sent"
```

Comment:
I don't think this factory is valid, since that's not one of the `VALID_STATUSES`